client/src/pages/Client/index.tsx

```diff
@@ -1,95 +1,220 @@
+                      $
+                      {job.budget}
+                    </span>
+                  </div>
+                </div>
+              </AccordionSummary>
+              <AccordionDetails>
+                {job.proposals.map((proposal) => <ProposalJob proposal={proposal} acceptProposal={acceptPropsal} />)}
+              </AccordionDetails>
+            </Accordion>
+
+          </JobCard>
+        ))}
+      </>
+    );
+  }
+  const tablist: Array<TabListInt> = [{
+    label: 'Pending',
+    child: pendingJobs,
+  },
+  { label: 'Accepted', child: acceptedJobs }];
   return (
-    <div>
-      <h1>
-        client
-        {id}
-      </h1>
-      <Button onClick={handleOpen}>Open modal</Button>
-      <JobForm showModel={showModel} handelClose={handleClose} />
-
-      <JobCard job={job}>
-        <Accordion disabled={!proposals.length}>
-          <AccordionSummary>
-            <div className="budget-proposal-section budget-proposal-client ">
-              <div className="proposals">
-                proposals:
-                <span>
-                  {job.proposals.length}
-                </span>
-              </div>
-              <div className="budget">
-                budget:
-                <span>
-                  $
-                  {job.budget}
-                </span>
-              </div>
-            </div>
-          </AccordionSummary>
-          <AccordionDetails>
-            <ProposalJob proposal={proposal} />
-            <ProposalJob proposal={proposal} />
-
-          </AccordionDetails>
-        </Accordion>
-      </JobCard>
+    <div className="client-pro container">
+      <div className="btn-table">
+        <Button
+          onClick={handleOpen}
+          style={{
+            fontSize: '12px',
+            borderRadius: '9px',
+            backgroundColor: '#132754',
+            color: '#fff',
+            width: '6%',
+            margin: '7px',
+          }}
+        >
+          Add
+
+        </Button>
+      </div>
+      <JobForm
+        showModel={showModel}
+        handelClose={handleClose}
+        setJobsUnoccupied={setJobsUnoccupied}
+        jobsUnoccupied={jobsUnoccupied}
+      />
+      <Tabs tablist={tablist} />
+      <Stack spacing={2} sx={{ width: '100%' }}>
+        <Snackbar
+          anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+          open={openAlert}
+          onClose={() => setOpenAlert(false)}
+          autoHideDuration={6000}
+        >
+          <Alert severity="success">
+            {message}
+          </Alert>
+        </Snackbar>
+      </Stack>
+      <Stack spacing={2} sx={{ width: '100%' }}>
+        <Snackbar
+          anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+          open={openAlertProposal}
+          onClose={() => setOpenAlert(false)}
+          autoHideDuration={6000}
+        >
+          <Alert severity="success">
+            {accept}
+          </Alert>
+        </Snackbar>
+      </Stack>
```

Comment:
No need to have 2 snackbars, as it a component I can control when to open and what the message would be